lib/danger/danger_core/plugins/dangerfile_github_plugin.rb

```diff
@@ -224,6 +224,16 @@ def html_link(paths, full_path: true)
       paths.first(paths.count - 1).join(", ") + " & " + paths.last
     end
 
+    # @!group Github Misc
+    # Tell github to ingore any inline message whihc is not in diff's range, not post as main comment.
+    # @param    [Bool] dismiss
+    #           Ingore out of range inline messages, defaults to `true`
+    #
+    # @return   [void]
+    def dismiss_out_of_range_messages(dismiss: true)
```

Comment:
since this method name is **dismiss**_out_of_range_messages, i think when someone calls it without any arguments, he will expect this to dismiss messages as the name's mean.
how do you think about this? maybe change to another name is better?